.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: pip
+    directories:
+      - /utils/cq/requirements.txt
+      - /requirements-build.txt
+      - /requirements-ftest.txt
+      - /requirements-utest.txt
```

Comment:
I'm not sure about this. The key is "directories" but I'm trying to include specific "files".
I think I might instead need
```suggestion
    directories:
      - /utils/cq/
      - /
```

Reply:
Oh, hrm.  Will Dependabot know about `requirements-*.txt` files?

Reply:
Apparently not :/ The file has to end with "requirements.txt"

Reply:
But I wonder if it will look at `/requirements.txt` and recursively drill down?